src/browser/service/storage.rs

```diff
@@ -34,20 +34,3 @@ where
         .set_item(name, &serialized)
         .expect("save into `LocalStorage`");
 }
-
-/// Load a store, to a deserializable data structure.
-#[deprecated(
-    since = "0.7.0",
-    note = "Use `LocalStorage::get` or `SessionStorage::get`."
-)]
-pub fn load_data<T>(storage: &Storage, name: &str) -> Option<T>
```

Comment:
Probably we don't have to delete this function now. I think it's out of scope of this PR and in the future the whole `service/storage` will be deleted.

Reply:
I'll bring it back in in my next commit.